mozilla_django_oidc/auth.py

```diff
@@ -110,8 +110,8 @@ def verify_token(self, token, **kwargs):
 
         # Verify the token
         verified_token = self._verify_jws(
-            token,
-            key,
+            force_bytes(token),
+            force_bytes(key),
```

Comment:
Actually, on second thought, perhaps this ought to be `smart_bytes` instead since it's possible it might come from `settings`.